### PR TITLE
[Relay] Fetch reputation history

### LINKED ISSUE #436: [UST-179] Fetch ClaimedReputationHistory API (Relayer)
**Is your feature request related to a problem? Please describe.**
add an api GET /api/reputation/history for @[Update User’s Reputation Page](https://www.notion.so/Update-User-s-Reputation-Page-7b5ae96d4ec847199ece3bc8f923e0e8?pvs=24)

**Describe the solution you'd like**
1. should take `from_epoch` and `to_epoch`
2. relayer call `db.find({epoch: {from: from_epoch, to: to_epoch}})`
3. return the history && status=`200`

### PR DESCRIPTION
**Pull request by Xiawpohr**
## Summary

Describe the changes made in your pull request here.

## Linked Issue

close #436 

## Details

1. should take `from_epoch` and `to_epoch`
2. relayer call `db.find({epoch: {from: from_epoch, to: to_epoch}})`
3. return the history && status=`200`

## Todo

Break down the progress of the PR for everyone to see what else you intend to include in this PR.

-   [x] Draft implementation for `/reputation/history`
-   [x] Unit tests for `/reputation/history`

## Checklist

-   [x] All new and existing tests pass
-   [x] I have updated the documentation (if applicable)
-   [x] My changes do not introduce new security vulnerabilities

### CODE REVIEW

File: packages/relay/test/reputation.test.ts [hunk 1]
```diff
@@ -0,0 +1,91 @@
+import { expect } from 'chai'
+import { ethers } from 'hardhat'
+
```
Comment (ryanycw) [inline packages/relay/test/reputation.test.ts:3]:
```suggestion
```

File: packages/relay/test/reputation.test.ts [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+        chainId = await unirep.chainid()
+    })
+
+    after(async function () {
+        await stopServer('reputation', snapshot, sync, express)
+    })
+
+    it('should be able to get reputation history by fromEpoch and toEpoch', async () => {
+        await insertReputationHistory(db)
```
Comment (ryanycw) [inline packages/relay/test/reputation.test.ts:69]:
```suggestion
        //TODO: Update here, use actual claim feature to insert reputation history 
        await insertReputationHistory(db)
```

**Review by ryanycw — APPROVED** [review]
🛳️

File: packages/relay/src/types/Reputation.ts [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import { ReportHistory } from './Report'
+
+export enum ReputationType {
```
[the remark this answers is not in the document]
Comment (Xiawpohr) [reply packages/relay/src/types/Reputation.ts:3]: Yes

**Review by ryanycw — APPROVED** [review]
🎇🎇🎇

**Pull request merged**